paper-ieee-trans.tex

```diff
@@ -315,15 +315,16 @@ \subsection{EGI--Integrated Sustainable Pan--European Infrastructure}
 was the continuation of the {\sl EGEE--III} project, with the objective of establishing and
 maintaining a sustainable European Grid Infrastructure, composed by a federation of National
 Grid Initiatives and interoperable with other Grids worldwide.
-This goal would be accomplished through the
-development and maintenance of operational tools, such as the Operations Portal \cite{egi-ops},
-the {\sl EGI} Helpdesk \cite{ggus}, or the Grid Operations Configuration DataBase (GOCDB) \cite{gocdb}, and the
-management of the software provisioning process \cite{mario}, to validate and distribute the software to
-the production infrastructure. As a result, this process led to a production--ready
+This has been accomplished through the
+development and maintenance of some operational tools, such as the Operations Portal \cite{egi-ops},
```

Comment:
"some" here seems that not all the operational tools were developed or maintained..although this could be true ;) I don't think it is needed here

Reply:
maybe "diverse" or "various"? I've put "some" because I thought the enumeration was not exhaustive, we are not listing all the operational tools.

Reply:
various

Reply:
I've used "some" because it  seems it is not an exaustive list of tools. Maybe "diverse" or "various"?

Reply:
adding various